packages/db/src/config.ts

```diff
@@ -33,24 +34,39 @@ export const getSecretConnectionConfig = async (
       throw new Error(`AWS Secret ${SecretId} is missing required key '${key}'`);
```

Comment:
Note for this function, not this line: could we add a docstring?

Reply:
We can - I'm less inclined to in Typescript generally if the function is self-explanatory as it's self-annotated already (typing is duplicative). 

That said: ea8ae4fd6d6

Reply:
Hmm yeah maybe if the arguments are (properly) typed we don't need a docstring. Descriptions are handy sometimes but also prone to going stale. And in this case, obvious.